### PR TITLE
contributing.md

### PR DESCRIPTION
**Pull request by sukanvisapearyoo**
docs(contributing): Modified the 'Get Started' section by adding iOS and android 'Install dependencies' part

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,17 @@
...
+   XCode
+   Node & Watchman using Homebrew.
+    - brew install node
+    - brew install watchman
+   Cocopod 
+    - $ sudo gem install cocoapods.
+
+   Andriod: 
+   Android development environment.
+   Java Development Kit - Zulu using Homebrew.
```
Comment (cdiddy77) [inline CONTRIBUTING.md:23]: this isn't necessary. These sorts of packages all assume that users have already set up a react native development environment.

**Review by cdiddy77 — COMMENTED** [review]
This change isn't necessary

**Comment by rorleth** [comment]
Also note: the "issue" work item complains about another issue.

**Review by cdiddy77 — CHANGES_REQUESTED** [review]
This change isn't really necessary -- as noted, the inserted lines are not needed. Should we close this PR or were there other changes you wanted to make?

**Comment by sukanvisapearyoo** [comment]
I will make changes. I got some answers from Robert.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,7 +24,7 @@ Read the READMEs in [`android/`](android/README.md) and [`ios/`](ios/README.md)
 
 ### iOS
 
-1. Open the `react-native-mediapipe/example/ios/MediapipeExample.xcworkspace` file with Xcode
+1. Open the `react-native-mediapipe/examples/ios/MediapipeExample.xcworkspace` file with Xcode
```
Comment (rorleth) [inline CONTRIBUTING.md:27]: That is still not the correct path - there is a folder called objectdetection under /examples/ and the xcworkspace file is called differently.

**Review by rorleth — APPROVED** [review]

**Pull request merged**